homeassistant/components/binary_sensor/ping.py

```diff
@@ -126,7 +129,18 @@ def ping(self):
                     'avg': rtt_avg,
                     'max': rtt_max,
                     'mdev': ''}
-
+            if 'max/' not in str(out):
+                _LOGGER.debug("Detected Alpine ping...")
```

Comment:
This doesn't add any value. There are the people who knows that they are running on Alpine and the ones who don't care.